database/src/migrations/20230726080300_clean_project_relations.ts

```diff
@@ -0,0 +1,63 @@
+import { Knex } from 'knex';
+
+const DB_SCHEMA = process.env.DB_SCHEMA;
+const DB_SCHEMA_DAPI_V1 = process.env.DB_SCHEMA_DAPI_V1;
+
+/**
+ * Remove caveats and climate relations
+ *
+ * @export
+ * @param {Knex} knex
+ * @return {*}  {Promise<void>}
+ */
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`
+    -------------------------------------------------------------------------
+    -- Remove old views
+    -------------------------------------------------------------------------
+    SET search_path = ${DB_SCHEMA_DAPI_V1};
```

Comment:
Doesn't really matter either way, but you could replace these two schema vars with the hardcoded strings.

The intended purpose never panned out for various reasons, so it doesn't matter if we hardcode them.

Reply:
I've got another PR that I'll replace these on